geode-wan/src/test/java/org/apache/geode/internal/cache/wan/wancommand/CreateGatewayReceiverCommandDUnitTest.java

```diff
@@ -76,18 +81,26 @@ public void testCreateGatewayReceiverWithDefault() {
       fail("testCreateGatewayReceiver failed as did not get CommandResult");
     }
 
+    String hostname;
+    try {
+      hostname = SocketCreator.getLocalHost().getHostName();
+    } catch (UnknownHostException e) {
```

Comment:
I think it's generally better practice to just declare that your test method throws Exception, rather than catching and rethrowing exceptions in your test.

Reply:
Good point.